src/insexpand.c

```diff
@@ -1249,6 +1249,12 @@ ins_compl_build_pum(void)
     if (compl_match_array == NULL)
 	return -1;
 
+    if (compl_started && compl_leader != NULL && *compl_leader == NUL)
+    {
+	compl_shown_match = compl_no_select ? compl_first_match
+					    : compl_first_match->cp_next;
+    }
+
```

Comment:
It seems that the message at the bottom is not updated.

Reply:
okay .. thanks